packages/edit-post/src/components/header/index.js

```diff
@@ -69,6 +70,10 @@ function Header() {
 						forceIsSaving={ isSaving }
 					/>
 				) }
+				<PreviewOptions
+					forceIsAutosaveable={ hasActiveMetaboxes }
+					forcePreviewLink={ isSaving ? null : undefined }
```

Comment:
What's the difference between `null` and `undefined` here?

Reply:
It's used to [set the preview link to null](https://github.com/WordPress/gutenberg/blob/7d2bf129410471a7c7023df55d5461ec97b4088a/packages/editor/src/components/post-preview-button/index.js#L233) if it's still saving.